README.md

```diff
@@ -123,7 +123,29 @@ This will ensure all event handlers will be registered rather being overruled by
 
 ### useRole
 
-(tbd)
+#### Usage
+
+```html
+<script>
+	import { useFloating, useInteractions, useRole } from '@skeletonlabs/floating-ui-svelte';
+
+	const floating = useFloating();
+
+	const role = useRole(floating.context, { role: 'tooltip' });
+
+	const interactions = useInteractions([role]);
+</script>
+
+<button {...interactions.getReferenceProps()}>Reference</button>
+<div {...interactions.getFloatingProps()}>Tooltip</div>
+```
+
+#### Options
+
+| Property | Description | Type | Default Value |
+| -------- | ----------- | ---- | ------------- |
+| enabled | Enables the interaction | boolean | true |
+| role | The role that the floating element should be | AriaRole | 'dialog' |
 
```

Comment:
We may want to link to the Floating UI docs for these two subsections. Seems like useful info that is not strictly speaking React-specific.

- https://floating-ui.com/docs/useRole#component-roles
- https://floating-ui.com/docs/useRole#native-roles